### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by Athight**
* email donationswap when an offer is confirmed
* set default offer expiration to ~3 months
* update some website text

### CODE REVIEW

File: src/donationswap.py [hunk 1]
```diff
@@ -552,6 +552,21 @@ def confirm_offer(self, secret):
...
+			'{%AMOUNT%}': offer.amount,
+			'{%MIN_AMOUNT%}': offer.min_amount,
+			'{%COUNTRY%}': offer.country.name
+		}
+
+		self._mail.send(
+			util.Template('email-subjects.json').json('post-confirmed-email'),
+			util.Template('post-confirmed-email.txt').replace(replacements).content,
+			html=util.Template('post-confirmed-email.html').replace(replacements).content,
+			to=self._config.contact_message_receivers['to']
```
Comment (Athight) [inline src/donationswap.py]: todolist item: Catherine would like an email to donationswap@eahub.org when a new offer is made (low priority)

File: src/static/start.js [hunk 2]
```diff
@@ -190,6 +190,19 @@
...
+			var year = info.today.year;
+
+			month += 3;
+			if (month > 12) {
+				month -= 12;
+				year += 1;
+			}
+
+			// month coming in and going out is 1-12, javascript Date takes 0-11 but day 0 gives last day of previous month
+			day = Math.min(day, new Date(year, month, 0).getDate());
```
Comment (Athight) [inline src/static/start.js]: 31 February is not ok!

File: src/static/start.js [hunk 3]
```diff
@@ -190,6 +190,19 @@
 			populateDropdowns(info);
 			attachEventHandlers();
 
+			var day = info.today.day;
+			var month = info.today.month;
+			var year = info.today.year;
+
+			month += 3;
```
Comment (Athight) [inline src/static/start.js]: todolist item: default offer expiration time: 1 year → 3 months

**Comment by Athight** [comment]
guess who forgot to merge their old pull request! 🙈

File: src/donationswap.py [hunk 4]
```diff
@@ -265,17 +272,25 @@ def _delete_unconfirmed_offers(self):
 		return count
 
 	def _send_mail_about_expired_offer(self, offer):
+		newExpirey = offer.expires_ts + (offer.expires_ts - offer.created_ts)
```
Comment (bigfighter) [inline src/donationswap.py]: expiry, not expirey. (oh, you kiwis... ;-) )

Comment (Athight) [reply src/donationswap.py]: english is hard 🙈

**Pull request merged**